core/nginx/start.py

```diff
@@ -2,6 +2,9 @@
 
 import os
 import subprocess
+from socrate import system
+
+system.set_env(log_filters=r'could not be resolved \(\d\: [^\)]+\) while in resolving client address, client\: [^,]+, server: [^\:]+\:(25,110,143,587,465,993,995)$')
```

Comment:
This filter must be removed. It is used in our fail2ban config for banning bots. 
If we still choose to remove this from the logs, then the the fail2ban rule in faq.rst that uses this log message, must be omitted.

Personally I prefer still having this filter. It is an easy method to ban bad behaving bots.

Reply:
Are you sure? This is the reverse DNS resolution failure line, not:

```
^\s?\S+ mailu\-front\[\d+\]: \S+ \S+ \[info\] \d+#\d+: \*\d+ client login failed: \"AUTH not supported\" while in http auth state, client: <HOST>, server:
```

Or are you saying that both get written in the same call and we should split/apply the regexp on each parts?

Reply:
I was wrong. It does work after all.